libse/Utilities.cs

```diff
@@ -2004,6 +2004,26 @@ public static string RemoveSpaceBeforeAfterTag(string text, string openTag)
             if (text.StartsWith(open1, StringComparison.Ordinal))
                 text = openTag + text.Substring(open1.Length);
 
+            string preText = string.Empty;
+            // fix spaces after quote...
+            // " In five years the Corleone family\r\nwill be completely legitimate."
+            if (text.LineStartsWithHtmlTag(true, true))
+            {
+                int endIdx = text.IndexOf('>') + 1;
+                preText = text.Substring(0, endIdx);
+                text = text.Substring(endIdx);
+            }
+            if (text.StartsWith('"'))
+            {
+                text = '"' + text.Substring(1).TrimStart();
+            }
+            text = preText + text;
+            // Foobar. " Foobar" => Foobar. "Foobar"
+            text = text.Replace(". \" ", ". \"");
+            text = text.Replace("? \" ", "? \"");
+            text = text.Replace("! \" ", "! \"");
+            text = text.Replace(") \" ", ") \"");

```

Comment:
This can actually be optimize to n instead of 3n, but lets keep it this way for now 😉 ...